### ISSUE TITLE
Dictionary feedback #2

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/argo-dictionary#150]
- [ ] Program ID note - change to:
This is the unique id that is assigned to your program.  If you have logged into the ARGO Data Platform, this is the Program Id that you see in the Program Services area. For example, PACA-CA is a Program ID.

**Comment by rosibaj** [comment]
Should the "type" for presenting_symptoms and treatment_type be "ARRAY" instead of "TEXT"?
@kcullion its an ARRAY of TEXT elements ( you can also have an array of intergers), so no on this.

**Comment by rosibaj** [comment]
Should we link to the new "Dictionary overview" page, or "Clinical data validation rules" page from the dictionary?  @kcullion can you move this as a ticket to the docs site? i think yes - can you define where to add that link for ui review?

**Comment by kcullion** [comment]
@rosibaj yep, made a ticket for me to do a quick proposal https://github.com/icgc-argo/argo-docs/issues/241

**Comment by rosibaj** [comment]
third point in #158

**Comment by rosibaj** [comment]
6ths point is already resolved

**Comment by rosibaj** [comment]
4th point address in pr #158

**Comment by rosibaj** [comment]
Tested on staging with dictionary 0.14, migrated to prod on dictionary 0.14
